app/helpers/mailer_helper.rb

```diff
@@ -9,6 +9,10 @@ def style_link(body, url, html_options = {})
     link_to(body.html_safe, url, html_options)
   end
 
+  def style_role(text)
+    tag.em(tag.strong("(#{text})"))
```

Comment:
Should the parentheses be included here? I know some languages use different parentheses, for example https://en.wikipedia.org/wiki/Chinese_punctuation#Marks_similar_to_European_punctuation, so extracting it to the locale files might be more translation-friendly